src/actors/rqgActorSheet.ts

```diff
@@ -223,6 +223,8 @@ export class RqgActorSheet extends ActorSheet<
       isEditable: this.isEditable,
       isGM: getGameUser().isGM,
       isPC: this.actor.hasPlayerOwner,
+      showCharacteristicRatings:
+        (getGame().settings.get(systemId, "showCharacteristicRatings") as boolean) || false,
```

Comment:
You don't need to cast the type if you register the type of the setting in https://github.com/sun-dragon-cult/fvtt-system-rqg/blob/eabfc02b40cb75b1614b664a2e5d3e8466a1ae91/src/global.d.ts#L47